hazelcast-spring/src/test/java/com/hazelcast/spring/TestFullApplicationContext.java

```diff
@@ -1,839 +1,879 @@
-        assertEquals(InMemoryFormat.OBJECT, queryCacheConfig.getInMemoryFormat());
-        assertTrue(queryCacheConfig.isCoalesce());
-        assertFalse(queryCacheConfig.isPopulate());
-        assertIndexesEqual(queryCacheConfig);
-        assertEquals("__key > 12", queryCacheConfig.getPredicateConfig().getSql());
-        assertEquals(EvictionPolicy.LRU, queryCacheConfig.getEvictionConfig().getEvictionPolicy());
-        assertEquals(EvictionConfig.MaxSizePolicy.ENTRY_COUNT, queryCacheConfig.getEvictionConfig().getMaximumSizePolicy());
-        assertEquals(111, queryCacheConfig.getEvictionConfig().getSize());
-    }
-
-    private void assertIndexesEqual(QueryCacheConfig queryCacheConfig) {
-        Iterator<MapIndexConfig> iterator = queryCacheConfig.getIndexConfigs().iterator();
-        while (iterator.hasNext()) {
-            MapIndexConfig mapIndexConfig = iterator.next();
-            assertEquals("name", mapIndexConfig.getAttribute());
-            assertFalse(mapIndexConfig.isOrdered());
-        }
-    }
-
-    @Test
-    public void testMapNativeMaxSizePolicy() {
-        MapConfig mapConfig = config.getMapConfig("map-with-native-max-size-policy");
-        MaxSizeConfig maxSizeConfig = mapConfig.getMaxSizeConfig();
-
-        assertEquals(MaxSizeConfig.MaxSizePolicy.USED_NATIVE_MEMORY_PERCENTAGE, maxSizeConfig.getMaxSizePolicy());
-    }
-
-    @Test
-    public void testHotRestart() {
-        File dir = new File("/mnt/hot-restart/");
-        HotRestartPersistenceConfig hotRestartPersistenceConfig = config.getHotRestartPersistenceConfig();
-        assertTrue(hotRestartPersistenceConfig.isEnabled());
-        assertEquals(dir.getAbsolutePath(), hotRestartPersistenceConfig.getBaseDir().getAbsolutePath());
-        assertEquals(1111, hotRestartPersistenceConfig.getValidationTimeoutSeconds());
-        assertEquals(2222, hotRestartPersistenceConfig.getDataLoadTimeoutSeconds());
-    }
-
-    @Test
-    public void testMapEvictionPolicyClassName() {
-        MapConfig mapConfig = config.getMapConfig("mapWithMapEvictionPolicyClassName");
-        String expectedComparatorClassName = "com.hazelcast.map.eviction.LRUEvictionPolicy";
-
-        assertEquals(expectedComparatorClassName, mapConfig.getMapEvictionPolicy().getClass().getName());
-    }
-
-    @Test
-    public void testMapEvictionPolicyImpl() {
-        MapConfig mapConfig = config.getMapConfig("mapWithMapEvictionPolicyImpl");
-
-        assertEquals(DummyMapEvictionPolicy.class, mapConfig.getMapEvictionPolicy().getClass());
-    }
-
-    @Test
-    public void testWhenBothMapEvictionPolicyClassNameAndEvictionPolicySet() {
-        MapConfig mapConfig = config.getMapConfig("mapBothMapEvictionPolicyClassNameAndEvictionPolicy");
-        String expectedComparatorClassName = "com.hazelcast.map.eviction.LRUEvictionPolicy";
-
-        assertEquals(expectedComparatorClassName, mapConfig.getMapEvictionPolicy().getClass().getName());
-    }
-
-
-
-}
+/*

```

Comment:
@dsukhoroslov There is too much change here. Looks like your second commit changed the line endings. Could you please change your line endings to LF instead of CRLF and commit again? (You can find the details here https://help.github.com/articles/dealing-with-line-endings)